### PR TITLE
EDU-1080 Extract filter component

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1080

**Before**
Screen Shot 2023-01-27 at 16 21 00

**Current**
Screen Shot 2023-01-27 at 16 28 00

### CODE REVIEW

File: src/components/pages/docs.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ newDocument:
 initialAuthor:
   en: Initial author
   de: Erstautor
-filterPlaceholder:
-  en: Filter by title or initial author
-  de: Nach Titel oder Erstautor filtern
+placeholder:
+  en: Search by title or initial author
+  de: Nach Titel oder Erstautor suchen
```
Comment (adrianaluchian) [inline src/components/pages/docs.yml:12]: Even though we use the input as a filter, we display to the user a search input, so I thought better to phrase this according to the input.

Comment (ahelmberger) [reply src/components/pages/docs.yml:12]: Is there a reason why you did not use the new component here as `<FilterInput size="large" ... />`?

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-1080
> 
> **Before** Screen Shot 2023-01-27 at 16 21 00
> 
> **Current** Screen Shot 2023-01-27 at 16 28 00

Looks actually a lot better than I anticipated :relaxed:

File: src/components/pages/docs.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/docs.yml:12]: The only reason is an invalid one actually 😆 I confused the "Documents" page with the "Search" page (not for the first time) and considered that here it should be search.
I shall change it!

Comment (adrianaluchian) [reply src/components/pages/docs.yml]: Changed, thanks!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**